bears/js/JSONFormatBear.py

```diff
@@ -10,10 +10,7 @@
 
 
 class JSONFormatBear(LocalBear):
-    try:
-        DecodeError = json.decoder.JSONDecodeError
-    except AttributeError:
-        DecodeError = ValueError
+    DecodeError = json.decoder.JSONDecodeError
```

Comment:
It does not ahttps://docs.python.org/3/library/json.html#json.JSONDecodeError - JSONDecodeError is new in 3.5
Hence it wont exist in 3.4 and 3.3. Which is why this is there.